application/src/main/java/com/sanction/thunder/authentication/Key.java

```diff
@@ -23,7 +25,7 @@ public Key(@JsonProperty("application") String application,
   }
 
   @JsonProperty("application")
-  public String getApplication() {
+  public String getName() {
     return application;
```

Comment:
We're required to call this method `getName()`, right? If so, then I think we should change this variable to be called `name` so that the method makes sense.